src/data/markdown/docs/02 javascript api/09 k6-http/10-asyncRequest- method- url- -body-- -params- -.md

```diff
@@ -0,0 +1,67 @@
+---
+title: 'asyncRequest( method, url, [body], [params] )'
+description: 'Issue any type of HTTP request asynchronously.'
+excerpt: 'Issue any type of HTTP request asynchronously.'
+---
+
+| Parameter         | Type                          | Description                                                                               |
+| ----------------- | ----------------------------- | ----------------------------------------------------------------------------------------- |
+| method            | string                        | Request method (e.g. `POST`). Must be uppercase.                         |
+| url               | string / [HTTP URL](/javascript-api/k6-http/urlurl#returns) | Request URL (e.g. `http://example.com`).                                                  |
+| body (optional)   | string / object / ArrayBuffer | Request body. Objects are `x-www-form-urlencoded`.                                    |
```

Comment:
@MattDodsonEnglish I did merge this without reading too thoroughly ... but while making a PR to fix it for `http.request` (where this is copied from) I did find it confusing.

```suggestion
| body (optional)   | string / object / ArrayBuffer | Request body. Objects will be `x-www-form-urlencoded` encoded.                                    |
```

Reply:
https://github.com/grafana/k6-docs/pull/1035